api/v1alpha1/virtualmachine_conversion.go

```diff
@@ -1078,6 +1078,7 @@ func (src *VirtualMachine) ConvertTo(dstRaw conversion.Hub) error {
 	restore_v1alpha3_VirtualMachineReadinessProbeSpec(dst, restored)
 
 	dst.Status = restored.Status
+	dst.Spec.BiosUUID = restored.Spec.BiosUUID
```

Comment:
To keep the same pattern can we have a `restore_v1alpha3_VirtualMachineBiosUUID()` function like we do above?

Reply:
Done 👍